script/Ciel/Day 13/13_01_01_CIEL13_3B.txt

```diff
@@ -1649,27 +1767,31 @@ Without uttering even the slightest sound―――
 [sha:c237adf237533067cdb6963172d61cf048bb9492]{
 -- Page 242, Offset 33878.
 -- シエルという代行者が、俺を殺す為に踏み込んできた。
-the Executor known as Ciel steps forward in order to kill me.
+The Executor known as Ciel steps forward in order to kill me.
 }
 [sha:d0e738f5bd5cf3b3ecc1ddd271351b32c48e5503]{
 -- Page 243, Offset 33879.
 -- もうこれ以上は語る言葉もなければ語る必要もないと。
-As if to say that there's no longer any need for words.
+She has no more words to say,
+ and there remains nothing that needs be said.
 }
 [sha:6a5ec3c4bdb450336607105365043e47571939c6]{
 -- Page 243, Offset 33880.
 -- 女の両腕が、あの“凶器”の引き金に当てられる。
-She places both of her arms on the trigger of her lethal weapon.
+// I don't know how you hit a trigger with your arms but if I see a CG where she's really doing that I'll come back and change it
+Her hands are wrapped around the trigger of her weapon.
 }
 [sha:8cbfc759dc71f9f1711ff606fa76e5d076c702ac]{
 -- Page 244, Offset 33881.
 -- 第七聖典は多くの死因をカタチにしたものだが、その本質は『魂砕き』にある。
-The Seventh Holy Scripture is made up of several causes of death given form, but its essence is that of 'soul crushing'.
+The Seventh Holy Scripture is made up of several causes of death given form,
+ but its underlying theme is 'Soul Crushing'.
```

Comment:
Should probably use "soul-shattering", sounds more accurate. Don't think it needs to be capitalized either. The "theme" part I don't quite see the point in changing. Essence can be easily reworded.

>The Seventh Holy Scripture is made up of several causes of death given form,
>but its essence lies in 'soul-shattering'.